map_model/src/make/initial/lane_specs.rs

```diff
@@ -263,29 +257,24 @@ pub fn get_lane_specs(tags: &Tags) -> Vec<LaneSpec> {
         need_back_shoulder = false;
     }
 
-    // TODO Now I'm regretting the weird order this returns...
     let mut specs = LaneSpec::normal(fwd_side, back_side);
     if need_fwd_shoulder {
-        let idx = specs
-            .iter()
-            .position(|s| s.reverse_pts)
-            .unwrap_or(specs.len());
+        specs.push(LaneSpec {
```

Comment:
Yep, because the order of `specs` is now left-to-right. The leftmost lane is the backwards lane farthest from the road center. So actually this change made this code easier -- a shoulder is on one edge of a road or the other.